packages/block-library/src/query/edit/index.js

```diff
@@ -104,6 +104,7 @@ export function QueryContent( {
 	return (
 		<>
 			<QueryInspectorControls
+				clientId={ clientId }
```

Comment:
We can skip plumbing this through for the `panelId` as the `ToolsPanel` and its items will be self-contained in this block's controls.